src/definition/users/IPostUserCreated.ts

```diff
@@ -0,0 +1,21 @@
+import { IHttp, IModify, IPersistence, IRead } from '../accessors';
+import { AppMethod } from '../metadata';
+import { IUserContext } from './IUserContext';
+
+/**
+ * Event interface that allows an app to
+ * register as a handler of the `IPostUserCreated`
+ * event
+ *
+ * This event is triggered *after* the
+ * user has been saved to the database.
+ */
+export interface IPostUserCreated {
+    [AppMethod.EXECUTE_POST_USER_CREATED](
+        context: IUserContext,
```

Comment:
Will this context have whether the user signed up themselves or if an administrator created the user?

Reply:
Yes. If some admin has created the user, the `performedBy` prop will be filled with the admin user, otherwise, the field will be null.